frontend/src/components/auth/UserManager.vue

```diff
@@ -61,16 +78,23 @@ export default {
       return Object.values(this.users);
     },
     bindArgs() {
-      const activeTab = this.$refs.userTab?.currentTab || 0;
-      if (activeTab === 1) {
-        return {recipientType: "User", recipientId: this.selectedUser.id};
-      }
-      return {user: this.selectedUser};
+      return [
+        { user: this.selectedUser },
+        { recipientType: "User", recipientId: this.selectedUser?.id },
+      ]
     }
   }
 }
 </script>
 
-<style scoped>
+<style lang="css" scoped>
+.users-list {
+  flex: 1;
+  min-width: 220px;
+}
 
-</style>
\ No newline at end of file
+.user-detail {
+  flex: 2;
+  min-width: 360px;
+}
```

Comment:
That is a good point, e.g. in case somebody has a real long username we would not like that to mess up the UI :+1: 

Yes, it looks like the Flex utility of Bootstrap does not provide that much flexibility.

Let's keep your current solution, which already works as intended :slightly_smiling_face: